src/components/SongItem/SongItem.tsx

```diff
@@ -31,6 +32,14 @@ const SongItem = ({ song }: SongProps) => {
         setConfirmationIsVisible(true)
     }
 
+    const upvote = () => {
+        dispatch(playlistsAsyncActions.vote({songId: song.id, voteType: VoteType.upVote, playlistType}))
```

Comment:
this two method is the same. instead you could have only one vote method, with a parameter(VoteType). and call it like onCilck={vote(VoteType.down/upvote)}